playwright/_impl/_transport.py

```diff
@@ -84,22 +84,21 @@ def __init__(
         self, loop: asyncio.AbstractEventLoop, driver_executable: Path
     ) -> None:
         super().__init__(loop)
-        self._stopped = False
+        self._stopped = asyncio.Event()
         self._driver_executable = driver_executable
 
     def request_stop(self) -> None:
-        self._stopped = True
+        self._stopped.set()
         self._output.close()
 
     async def wait_until_stopped(self) -> None:
-        await self._stopped_future
+        await self._stopped.wait()
```

Comment:
Don't you think it is confusing about what does `_stopped` do and `_stopped_future` previously ?

Reply:
With this change there is only `_stopped` which is a lot simple to reason about